packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.tsx

```diff
@@ -852,6 +852,10 @@ export class SearchInWorkspaceResultTreeWidget extends TreeWidget {
             mode: 'reveal'
         } : undefined;
 
+        if (preview) {
+            opts = Object.assign(opts || {}, { preview });
+        }
+
         const editorWidget = await this.editorManager.open(fileUri, opts);
```

Comment:
@danarad05 in order to get the feature to work I also had to update the following to determine the best handler instead of directly calling the `editorManager`.

```ts
const opener = await this.openerService.getOpener(fileUri, opts);
const editorWidget = opener.open(fileUri, opts) as EditorWidget;
```

For the following cases, the **opener** will be (based on the `canHandle` for the opener):
- `preview = false` = `EditorWidget`
- `preview = true` =  `EditorPreviewWidget`

There may be additional work to be done to get the `doubleClick` from the tree to convert the editor from `preview` to regular (similarly to the explorer).

Reply:
Thanks @vince-fugnitto, updated it as you described and supported also `doubleClick `to open without preview.